docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -36,130 +41,78 @@ log data has the following data retention guidelines:
+backing indices for these data streams.
 
-To view the default policy in {kib}:
+To view the `logs` policy in {kib}:
 
-. Go to Management and select *Index Lifecycle Policies*.
-. Search for _filebeat_
-. Select the _filebeat-version_ policy.
+. Open the menu and go to **Stack Management > Index Lifecycle Policies**.
+. Select the `logs` policy.
 
-This policy initiates the rollover action when the index size reaches 50GB or
-becomes 30 days old.
+The `logs` policy uses our recommended rollover defaults. These defaults
+initiate a rollover when the current write index reaches 50GB or becomes 30 days
+old.
 
-[role="screenshot"]
-image::images/ilm/tutorial-ilm-hotphaserollover-default.png["Default policy"]
+To confirm these rollover settings, click **Settings** for the hot phase. Then
+disable **Use recommended defaults** to display the rollover settings.
 
+[role="screenshot"]
+image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]
 
 [discrete]
+[[ilm-ex-modify-policy]]
 ==== Modify the policy
 
-The default policy is enough to prevent the creation of many tiny daily indices.
-You can modify the policy to meet more complex requirements.
+The `logs` policy's default settings are enough to prevent the creation of many
+tiny daily indices. You can modify the policy to meet more complex requirements.
 
-. Activate the warm phase.
+. Activate the warm phase and click **Settings**.
 +
 --
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png["Modify to add warm phase"]
-
-.. Set one of the following options to control when the index moves to the warm phase:
-
-*** Provide a value for *Timing for warm phase*. Setting this to *15* keeps the
-indices on hot nodes for a range of  15-45 days, depending on when the initial
-rollover occurred.
+image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png[Add a warm phase with custom settings]
 
-*** Enable *Move to warm phase on rollover*. The index might move to the warm phase
-more quickly than intended if it reaches the *Maximum index size* before the
-the *Maximum age*.
+.. Set *Move data into phase when* to *30 days*. This keeps indices in the hot
+tier for 30 days. After 30 days, indices move to the warm tier.
 
-.. In the *Select a node attribute to control shard allocation* dropdown, select
-*data:warm(2)* to migrate shards to warm data nodes.
+.. Enable **Replicas** and change *Number of replicas* to *1*.
 
-.. Change *Number of replicas* to *1*.
-
-.. Enable *Force merge data* and set *Number of segments* to *1*.
-
-NOTE:  When rollover is enabled in the hot phase, action timing in the other phases
-is based on the rollover date.
+.. Enable *Force merge* and set *Number of segments* to *1*.
 --
 
-. Activate the delete phase and set *Timing for delete phase* to *90* days.
+. Activate the delete phase and set *Move data into phase when* to *90* days.
```

Comment:
Might be worth clarifying here, too. "This moves indices to the delete phase 90 days after rollover."